include/agencies.csv

```diff
@@ -83,7 +83,7 @@ Nuclear Regulatory Commission,NRC
 Occupational Safety and Health Review Commission,OSHRC
 Office of Government Ethics,OGE
 Office of Personnel Management,OPM
-Overseas Private Investment Corporation,OPIC
+Overseas Private Investment Corporation,DFC
```

Comment:
Can probably just remove line 86 since, after discussing with @jsf9k, we are going to wait for GSA to update the agency name on https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv.